Taskfile.yml

```diff
@@ -268,24 +269,24 @@ tasks:
       - echo 'documentation served from http://localhost:8080/'
       - godoc -http=localhost:8080
 
-  outdated:
+  dependency:outdated:
     desc: list directly dependent modules that can be upgraded
     cmds:
       - go list -u -m $(go list -m -f '{{`{{.Indirect}} {{.}}`}}' all | grep '^false' | cut -d ' ' -f2) | grep '\['
 
-  graph:
+  dependency:graph:
     desc: graph of upstream modules with gmchart
     cmds:
       - go install github.com/PaulXu-cn/go-mod-graph-chart/gmchart@latest
       - go mod graph | gmchart
 
-  licenses:
+  dependency:licenses:
     desc: list project dependency licenses
     cmds:
       - go install github.com/google/go-licenses@latest
       - go-licenses report .
 
-  binmap:
+  dependency:binmap:
```

Comment:
Perhaps total navel gazing but binmap is not strictly for dependencies

Reply:
You're right, half the data it produces is probably related to the go runtime 😄 I'll revert it